FsSonarRunner/zip-assembly.xml

```diff
@@ -8,13 +8,12 @@
   <includeBaseDirectory>false</includeBaseDirectory>
   <fileSets>
     <fileSet>
-      <directory>FsSonarRunner/bin/${configuration}/net472</directory>
+      <directory>FsSonarRunner/publish</directory>
       <outputDirectory>\</outputDirectory>
       <includes>
-        <include>*.dll</include>
-        <include>*.exe</include>
-        <include>*.config</include>
-        <include>*/FSharp.Core.resources.dll</include>
+        <include>win-x86/**</include>
```

Comment:
Plugin is now quiet large - about 100MB. That's because of 3 self-contained dotnet runners.
On out server the next largest is Cfamily-plugin with only 27MB.

Reply:
Either require users to have dotnet core installed or have different budles depending on OS.

Reply:
Now configured it as Framework-dependent executables (FDE). This includes all except the dotnet core runtime.

Reply:
jar file is about 20MB for all 3 operating systems

Reply:
reverted